src/components/features/StatusScreen.tsx

```diff
@@ -190,20 +184,30 @@ const StatusScreen: FC = () => {
                     />
 
                     <NumberInput
-                        ref={maxVolumeInputRef}
-                        label="Maximum amplifier volume"
-                        description="From 0 to 100, step is 1"
+                        ref={volumeLimitInputRef}
+                        label="Volume limit"
+                        description="Prevents dangerous volumes"
```

Comment:
I like how this new `description` gets at the "why" rather than the "what" of the old `description`. That's much better. Nice!